ibis-server/app/model/__init__.py

```diff
@@ -48,23 +50,23 @@ class QueryTrinoDTO(QueryDTO):
 class BigQueryConnectionInfo(BaseModel):
     project_id: str
     dataset_id: str
-    credentials: str = Field(description="Base64 encode `credentials.json`")
+    credentials: SecretStr = Field(description="Base64 encode `credentials.json`")
 
 
 class ClickHouseConnectionInfo(BaseModel):
     host: str
     port: int
     database: str
     user: str
-    password: str
+    password: SecretStr
 
 
 class MSSqlConnectionInfo(BaseModel):
     host: str
     port: int
     database: str
     user: str
-    password: str
+    password: SecretStr
```

Comment:
I think we should mask all info in the connection info. People might not want to expose their host, port, or any information that could make others try to access their machine.

Reply:
Sure